packages/components/src/types/props/pagination-bottom.ts

```diff
@@ -0,0 +1,18 @@
+import { Generic } from '@a11y-ui/core';
+import { watchBoolean } from '../../utils/prop.validators';
+
+/* types */
+/** de
+ * Setzt die Pagination unter die Tabelle.
+ */
+/** en
+ * Puts the Pagination below the table.
+ */
+export type PropPaginationBottom = {
+	paginationBottom: boolean;
```

Comment:
Bitte kein Property einführen, dass soll über das Theme entschieden werden.

Worse is better - Die API soll so unverfänglich wie möglich bleiben/werden. Kaum Styling über API - Ausnahme Variant.

Dafür sollen Themes genutzt werden. Lieber weniger Features.

Reply:
Einfach nur im DOM oben.